linux_os/guide/services/openstack/keystone/keystone_lockout_failure_attempts.var

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+title: 'Maximum Number of Failed Authentication Attempts'
+
+description: |-
+    Specifies the maximum number of failed authentication attempts allowed
+    prior to disabling a Keystone user.
+
+type: string
+
+operator: less than or equal
```

Comment:
Is this a correct var operator? Where do I find docs?

Reply:
You may be mistaking XCCDF for OVAL. The operator for the variable should be in test, no?

Reply:
I have confirmed that this is an issue in the yaml parser, I am working on a fix.

Reply:
https://github.com/OpenSCAP/scap-security-guide/pull/2900 fixes this, review would be great

Reply:
#2900 has been merged, this needs a rebase and then the operator will work. it uses the same values XCCDF allows.